README.rst

```diff
@@ -100,8 +100,6 @@ The ``--config`` flag points to a ``.yaml`` file that is responsible to configur
 `configuration.yaml <https://github.com/input-output-hk/cardano-node/blob/master/configuration/defaults/byron-mainnet/configuration.yaml>`_.
 Some of the more important settings are as follows:
 
-* ``NodeId: 0``  -- Used in mock protocols only to differentiate nodes.
```

Comment:
NodeId was only used for mock protocols, I removed it wherever I could.

(We have way too many scripts, probably horribly broken).